certified-connectors/Oneflow/apiDefinition.swagger.json

```diff
@@ -1223,117 +1261,101 @@
               "type": "string",
-              "description": "notice_period",
-              "x-ms-summary": "Contract Notice Period"
+              "x-ms-summary": "Contract notice period"
             },
             "start_date": {
               "type": "string",
-              "description": "start_date",
-              "x-ms-summary": "Contract Start Date"
+              "x-ms-summary": "Contract start date"
             },
             "type": {
               "type": "string",
-              "description": "type",
+              "description": "no_duration/single_period/recurring/recurring_two_periods",
               "x-ms-summary": "Contract Type"
             }
           },
-          "description": "lifecycle_settings"
+          "description": "lifecycle_settings attributes"
         },
         "lifecycle_state": {
-          "x-ms-visibility": "internal",
-          "x-ms-summary": "Lifecycle state",
           "type": "object",
           "properties": {
             "contract_end_time": {
               "type": "string",
-              "description": "contract_end_time",
-              "x-ms-summary": "Contract End Time"
+              "x-ms-summary": "Contract end time"
             },
             "contract_start_time": {
               "type": "string",
-              "description": "contract_start_time",
-              "x-ms-summary": "Contract Start Time"
+              "x-ms-summary": "Contract start time"
             },
             "cancel_time": {
               "type": "string",
-              "description": "cancel_time",
-              "x-ms-summary": "Contract Cancel Time"
+              "x-ms-summary": "Contract cancel time"
             },
             "has_ended_by_termination": {
               "type": "boolean",
-              "description": "has_ended_by_termination",
-              "x-ms-summary": "Contract Ended By Termination?"
+              "x-ms-summary": "Contract ended by termination?"
             },
             "has_passed_notice_period_start_time": {
               "type": "boolean",
-              "description": "has_passed_notice_period_start_time",
-              "x-ms-summary": "Has Contract Passed Notice Period Start Time?"
+              "x-ms-summary": "Has notice period start time passed?"
             },
             "is_canceled": {
               "type": "boolean",
-              "description": "is_canceled",
-              "x-ms-summary": "Is Contract Canceled"
+              "x-ms-summary": "Is contract canceled"
             },
             "is_recurring": {
               "type": "boolean",
-              "description": "is_recurring",
-              "x-ms-summary": "Is Contract Recurring"
+              "x-ms-summary": "Is contract recurring"
             },
             "lifecycle_state": {
               "type": "string",
-              "description": "lifecycle_state",
-              "x-ms-summary": "Lifecycle State"
+              "description": "awaiting/active/ended",
```

Comment:
@DmitryKleshchov please make more detailed descriptions if you want connector users to choose between possible values. You can refer to the [guidelines for creating strings](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) or [coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters).